frontend/src/metabase-lib/metadata/Database.ts

```diff
@@ -40,6 +40,16 @@ class DatabaseInner extends Base {
   // Only appears in  GET /api/database/:id
   "can-manage"?: boolean;
 
+  constructor(object) {
+    super(object);
+
+    if (window.IS_STORYBOOK) {
+      for (const property in object) {
+        this[property] = object[property];
+      }
+    }
+  }
+
```

Comment:
Not sure we should make `metabase-lib` dependant on Storybook or `window`
What are you trying to achieve here?